### PR TITLE
Create fama23

### PR DESCRIPTION
**Pull request by ghost**
asd

### CODE REVIEW

**Comment by ghost** [comment]
ewr

File: fama23 [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+Use this section to tell people about which versions of your project are
+currently being supported with security updates.
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 5.1.x   | :white_check_mark: |
+| 5.0.x   | :x:                |
+| 4.0.x   | :white_check_mark: |
+| < 4.0   | :x:                |
```
Comment (ghost) [inline fama23:13]: 4AEE18F83AFDEB23

**Comment by ghost** [comment]
fg

File: fama23 [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Security Policy
+
+## Supported Versions
+
+Use this section to tell people about which versions of your project are
+currently being supported with security updates.
+
+| Version | Supported          |
+| ------- | ------------------ |
```
Comment (fama23) [inline fama23:9]: 4AEE18F83AFDEB23

**Review by ghost — COMMENTED** [review]
sdg

File: fama23 (hunk 1 above)
Comment (ghost) [inline fama23:13]: 4AEE18F83AFDEB23

**Review by ghost — COMMENTED** [review]
12

File: fama23 [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# Security Policy
+
```
Comment (ghost) [inline fama23:2]: 4AEE18F83AFDEB23

**Review by ghost — COMMENTED** [review]
11123

**Comment by ghost** [comment]
asr

File: fama23 (hunk 2 above)
Comment (ghost) [inline fama23:9]: 4AEE18F83AFDEB23

**Review by ghost — COMMENTED** [review]
123

**Comment by ghost** [comment]
ret

**Pull request closed without merging**